### PR TITLE
[Footer] Mentions that the site represents the undergraduate sub-committee

### PR DESCRIPTION
**Pull request by julianguyen**
Updated footer

### CODE REVIEW

File: themes/notmyidea/templates/base.html [hunk 1]
```diff
@@ -71,10 +71,10 @@
 
         <footer id="contentinfo" class="body">
                 <address id="about" class="vcard body">
-                Proudly powered by <a href="http://getpelican.com/">Pelican</a>, which takes great advantage of <a href="http://python.org">Python</a>.
+                    Women in Computer Science (WiCS) is the undergraduate sub-committee of the University of Waterloo Women in Computer Science Committee. Check out their <a href="https://cs.uwaterloo.ca/wics" target="_blank">website</a>!
```
Comment (ehashman) [inline themes/notmyidea/templates/base.html]: I would make this "Women in Computer Science Undergraduate (WiCS ugrad)" maybe?

Also, don't think `target="_blank"`is necessary on the link; at minimum, it's inconsistent with the rest of the site.

**Comment by ehashman** [comment]
Thanks for putting this together, Julia :) Awaiting your response on feedback and then I'll merge.

**Comment by julianguyen** [comment]
Updated

**Comment by julianguyen** [comment]
Can someone merge this?

**Comment by ehashman** [comment]
Can you please rebase your commits first? Having three commits for a 1 line change clogs up the log. e.g.

```
git rebase -i HEAD~3
```

which will take you to an editor asking for commands. Pick one commit and change the rest to "f" or "fixup".

**Comment by julianguyen** [comment]
I get the error: `Cannot 'fixup' without a previous commit`

**Comment by ehashman** [comment]
The top commit on the list has to be pick-ed (i.e. you're squishing them all down on top of it).

‎
  Original Message  
From: Julia Nguyen
Sent: Sunday, March 1, 2015 5:17 PM
To: wics-uw/website
Reply To: wics-uw/website
Cc: Elana Hashman
Subject: Re: [website] [Footer] Mentions that the site represents the undergraduate sub-committee (#43)

I get the error: Cannot 'fixup' without a previous commit

**Comment by julianguyen** [comment]
I set pick for the top commit and fixup for the bottom two but I get `error: could not apply c66a229... [Footer] Mentions that the site represents the undergraduate sub-committee`

**Comment by ehashman** [comment]
Please check IRC; let's try to pair on this remotely there.
‎‎

  Original Message  
From: Julia Nguyen
Sent: Sunday, March 1, 2015 5:49 PM
To: wics-uw/website
Reply To: wics-uw/website
Cc: Elana Hashman
Subject: Re: [website] [Footer] Mentions that the site represents the undergraduate sub-committee (#43)

I set pick for the top commit and fixup for the bottom two but I get error: could not apply c66a229... [Footer] Mentions that the site represents the undergraduate sub-committee

**Pull request closed without merging**